lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -12,5 +12,41 @@ def fulfillments(options = {})
       fo_fulfillments = get(:fulfillments, options)
       fo_fulfillments.map { |fof| FulfillmentOrderFulfillment.new(fof.as_json) }
     end
+
+    def move(new_location_id:)
+      body = { fulfillment_order: { new_location_id: new_location_id } }.to_json
+      keyed_fos = load_keyed_attributes_from_response(post(:move, {}, body))
+      if keyed_fos&.fetch('original_fulfillment_order', nil)&.attributes
+        load(keyed_fos['original_fulfillment_order'].attributes, false, true)
```

Comment:
just for my personal comprehension here but what are you trying to achieve here ? Specifically why does this check only apply to the `original_fulfillment_order` ?

Are you using it to determine wether or not we got here with the `error message` payload ?

Reply:
I was a bit uncertain about this one. The request is made using an instance of a FulfillmentOrder, e.g. `fulfillment_order.move(...)` where the server responds with multiple keyed FOs. I took the one named `original_fulfillment_order` to represent the mutation of the `fulfillment_order` that was the subject of the `.move` and so load the attributes of `original_fulfillment_order` into itself.

This is following the pattern where responses that return a single FO and calling `load_attributes_from_response` will load the calling object's attrributes from the only keyed value.